dashboard/config/locales/scripts.en.yml

```diff
@@ -7,20 +7,27 @@ en:
         csd_category_name: CS Discoveries ('17-'18)
         csd_2018_category_name: CS Discoveries ('18-'19)
         csd_2019_category_name: CS Discoveries ('19-'20)
+        csd_2020_category_name: CS Discoveries ('20-'21)
+        csd_pilot_category_name: CS Discoveries Pilot
         csf_category_name: CS Fundamentals
-        csf_2018_category_name: CS Fundamentals (2018)
-        csf_2019_category_name: CS Fundamentals (2019)
```

Comment:
I cleaned up some of the existing strings to make them consistent with our new format.  Is it ok that I change existing strings in this file?

Reply:
this is fine from the perspective of the i18n pipeline, but it might be worth double-checking with the curriculum team from a content perspective